packages/patrol_cli/lib/src/crossplatform/app_options.dart

```diff
@@ -135,7 +136,7 @@ class IOSAppOptions {
 
   /// Translates these options into a proper flutter build invocation, which
   /// runs before xcodebuild and performs configuration.
-  List<String> toFlutterBuildInvocation() {
+  List<String> toFlutterBuildInvocation(BuildMode buildMode) {
```

Comment:
Do you accept `buildMode` for the future? You don't use it now

Reply:
thanks, fixed